spinalcordtoolbox/deepseg/inference.py

```diff
@@ -342,6 +374,7 @@ def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
 
 
 def segment_totalspineseg(path_img, tmpdir, predictor, device, step1_only=False):
+    # Save probabilities is not used in totalspineseg, but we include it in the function signature to match the other segmentation functions
```

Comment:
I think we can remove this comment? Since, we're not actually modifying the `segment_totalspineseg` function signature (thankfully!).